src/main/java/com/cleanroommc/modularui/screen/Tooltip.java

```diff
@@ -81,7 +81,7 @@ public void draw(GuiContext context, @Nullable ItemStack stack) {
         int mouseX = context.getAbsMouseX(), mouseY = context.getAbsMouseY();
         IconRenderer renderer = IconRenderer.SHARED;
         List<String> textLines = lines.stream().filter(drawable -> drawable instanceof IKey).map(key -> ((IKey) key).get()).collect(Collectors.toList());
-        RenderTooltipEvent.Pre event = new RenderTooltipEvent.Pre(stack, textLines, mouseX, mouseY, screen.width, screen.height, thimaxWidth, TextRenderer.getFontRenderer());
+        RenderTooltipEvent.Pre event = new RenderTooltipEvent.Pre(stack, textLines, mouseX, mouseY, screen.width, screen.height, maxWidth, TextRenderer.getFontRenderer());
```

Comment:
that was supposed to be `this.maxWidth`, but had incredible skill issue during merge